### ISSUE TITLE
We need more puffin facts

### ISSUE THREAD

**Issue by nicole-a-tesla** [stuartlynn/puffin_facts#3]
There are only about 10 facts. This makes for a pretty boring site! We need 20 more facts.

**Comment by charlottemach** [comment]
http://mentalfloss.com/article/574766/puffin-facts

**Comment by Omabekee** [comment]
Hi. Can I contribute to this issue?

**Comment by nicole-a-tesla** [comment]
@Omabekee Absolutely

**Comment by nicole-a-tesla** [comment]
@Omabekee @charlottemach Do either of you want to take ownership of this project? I made this repo as an example for a conference workshop ~7 years ago, and it's very much not actively maintained! I'd be happy to transfer it if you're interested 😃

**Comment by Omabekee** [comment]
@nicole-a-tesla Certainly! The project piqued my interest and it seems like a fantastic chance to revive something that was originally intended as a conference example. I'd like to know more about the project's objectives and current status. Please feel free to provide more details, and let's talk about how might assume ownership and give it the attention it requires. Many thanks!
